src/styles/blocks/_reset.scss

```diff
@@ -0,0 +1,7 @@
+* {
```

Comment:
reset.scss is not a block, its more utility than block)
_Also blocks names doesnt starts with the `_`_